src/ICREATE3Factory.sol

```diff
@@ -3,26 +3,20 @@ pragma solidity >=0.6.0;
 
 /// @title Factory for deploying contracts to deterministic addresses via CREATE3
 /// @author zefram.eth
```

Comment:
Same.

Reply:
Should we keep the `@author zefram.eth` because of the AGPL-3.0 License?